packages/eas-cli/src/commands/env/exec.ts

```diff
@@ -103,6 +111,29 @@ export default class EnvExec extends EasCommand {
     };
   }
 
+  private async runCommandNonInteractiveWithEnvVarsAsync({
+    command,
+    environmentVariables,
+  }: {
+    command: string;
+    environmentVariables: Record<string, string>;
+  }): Promise<void> {
+    try {
+      const { stdout } = await spawnAsync('bash', ['-c', command], {
+        env: {
+          ...process.env,
+          ...environmentVariables,
+        },
+      });
+      Log.log(stdout);
```

Comment:
I believe that this will wait until command execution is completed and then log the whole command output all at once. I believe it's better to print logs as they appear (as we do in `--non-interactive` mode) to indicate to the user that the command execution is not stuck.